EventsExpress/ClientApp/src/containers/users.js

```diff
@@ -8,16 +8,18 @@ import UsersFilterWrapper from '../containers/user-filter';
 class UsersWrapper extends Component {
     componentDidMount() {
         this.getUsers(this.props.location.search);
+        this.props.get_count();
+        this.props.initialConnection();
```

Comment:
There is no place where you close connection. Consider doing it on component unmount.